queue/python/queue.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+Queue = {'front': None, 'back': None}
+
+
+class Node:
+    def __init__(self, data, next):
+        self.data = data
+        self.next = next
+
+
+    def Enqueue(Queue, element):
+        N = Node(element, None)
+        if Queue['back'] is None:
+            Queue['front'] = N
+            Queue['back'] = N
+        else:
+            Queue['back'].next = N
+            Queue['back'] = Queue['back'].next
+
+
+    def Dequeue(Queue):
+        if Queue['front'] is not None:
+            first = Queue['front']
+            Queue['front'] = Queue['front'].next
+            return first.data
+        else:
+            if Queue['back'] is not None:
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpduneabze/queue/python/queue.py
+++ b/tmp/tmpduneabze/queue/python/queue.py
@@ -17,7 +17,6 @@
         else:
             Queue['back'].next = N
             Queue['back'] = Queue['back'].next
-
 
     def Dequeue(Queue):
         if Queue['front'] is not None:
```